project/NativeImage.scala

```diff
@@ -122,7 +151,7 @@ object NativeImage {
         quickBuildOption ++
         debugParameters ++ staticParameters ++ configs ++
         Seq("--no-fallback", "--no-server") ++
-        Seq("--initialize-at-build-time=") ++
```

Comment:
This option was deprecated. Moreover, it has never included all the classes for build time initialization. In fact, it seems that the effect of this option was a no-op. At least for the GraalVM native-image since version 21.3.0.